app/src/main/java/org/fcitx/fcitx5/android/data/theme/CustomThemeSerializer.kt

```diff
@@ -55,6 +55,25 @@ object CustomThemeSerializer : JsonTransformingSerializer<Theme.Custom>(Theme.Cu
     private val strategies: List<MigrationStrategy> =
         // Add migrations here
         listOf(
+            MigrationStrategy("2.1") {
+                JsonObject(it.toMutableMap().apply {
+                    if (get("candidateTextColor") != null) {
+                        put("candidateTextColor", getValue("candidateTextColor"))
+                    } else {
+                        put("candidateTextColor", getValue("keyTextColor"))
+                    }
+                    if (get("candidateLabelColor") != null) {
+                        put("candidateLabelColor", getValue("candidateLabelColor"))
+                    } else {
+                        put("candidateLabelColor", getValue("keyTextColor"))
+                    }
+                    if (get("candidateCommentColor") != null) {
+                        put("candidateCommentColor", getValue("candidateCommentColor"))
+                    } else {
+                        put("candidateCommentColor", getValue("altKeyTextColor"))
+                    }
```

Comment:
```suggestion
                    put("candidateTextColor", getValue("keyTextColor"))
                    put("candidateLabelColor", getValue("keyTextColor"))
                    put("candidateCommentColor", getValue("altKeyTextColor"))
```

Filling all the missing theme properties from 2.0 to 2.1 should be enough. If a theme happens to have those properties, it should be versioned as 2.1